### PR TITLE
Add RA announcement to homepage and about page.

### PR DESCRIPTION
**Pull request by mikelehen**

### CODE REVIEW

File: src/screens/About/About.tsx [hunk 1]
```diff
@@ -21,27 +21,31 @@ export const sidebarItems: TocItem[] = [
     items: [
       {
         to: '/about#mission',
-        label: 'Our mission',
```
Comment (mikelehen) [inline src/screens/About/About.tsx:24]: I can undo these changes if we're not a fan, but all the text matched exactly so I figured I might as well remove the redundancy.

File: src/screens/About/About.tsx [hunk 2]
```diff
@@ -103,6 +107,11 @@ const About = () => {
           <MarkdownContent source={aboutContent.teamIntro} />
           <TeamSection />
         </HashWrapper>
+        <HashWrapper id="future-projects">
+          <AboutHeading2>{aboutContent.futureProjectsHeader}</AboutHeading2>
+          <MarkdownContent source={aboutContent.futureProjectsContent} />
+          <TeamSection />
```
Comment (pnavarrc) [inline src/screens/About/About.tsx:113]: The Team section is duplicated here, everyone shows up twice in the page

Comment (mikelehen) [reply src/screens/About/About.tsx:113]: oh, yikes.  Good catch.

**Review by pnavarrc — APPROVED** [review]
Looks good!

**Pull request merged**